app/controllers/users_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class UsersController < AuthenticationController
+  before_action :set_user, only: %i[edit update]
+  before_action :validate_current_user
+
+  def new
+    @user = current_local_authority.users.new
+  end
+
+  def create
+    @user = current_local_authority.users.new(user_params)
+
+    if @user.save
+      flash[:alert] = I18n.t("users.index.successfully_created")
+      redirect_to users_path
+    else
+      render :new
+    end
+  end
+
+  def index
+    @users = current_local_authority.users
+  end
+
+  def edit; end
+
+  def update
+    if @user.update(user_params)
+      flash[:alert] = I18n.t("users.index.successfully_updated")
+      redirect_to users_path
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def user_params
+    params
+      .require(:user)
+      .permit(:name, :email, :password, :mobile_number, :role)
+      .tap { |p| p.delete(:password) if p[:password].blank? }
```

Comment:
In the edit form, the password is not rendered. This means that it passes `password: ""` in the params, although obviously the admin isn't trying to set the password to ""!